web/php/Modules/Account/PM/PMDraftsModule.php

```diff
@@ -1,55 +1,52 @@
 <?php
 
-namespace Wikidot\Modules\Account\PM;
-
-
+declare(strict_types=1);
 
+namespace Wikidot\Modules\Account\PM;
 
-use Ozone\Framework\Database\Criteria;
-use Wikidot\DB\PrivateMessagePeer;
 use Wikidot\Utils\AccountBaseModule;
+use Wikijump\Models\UserMessage;
 
+/**
+ * AJAX Module for per-page view of drafts.
+ * @package Wikidot\Modules\Account\PM
+ */
 class PMDraftsModule extends AccountBaseModule
 {
 
+    /**
+     * Build drafts view.
+     * @param $runData
+     */
     public function build($runData)
     {
+        /** Not sure if this is worth breaking out into a setting. */
+        $per_page_limit = 30;
```

Comment:
Presumably later we'll have more of a user settings system and can do it then?

I think the main issue with wikidot limits is they're wholly arbitrary. People wouldn't mind fixed limits much if they were standard, as in they were always 10 for "big" things, 50 for "short" things (like messages or drafts), or something like that.